extensions/notebook/package.json

```diff
@@ -203,6 +208,10 @@
 				{
 					"command": "jupyter.cmd.managePackages",
 					"when": "false"
+        },
+        {
+					"command": "books.sqlserver2019",
+          "when": "sqlserver2019 && notebookQuality != stable"
```

Comment:
This doesn't seem very discoverable and will get messy as more books are added. For the command pallet part why not just have it so the command is just books.openBook and then if executed from the command pallet it shows a list of the available books to launch that the user can choose from? You can use the showQuickPick for this behavior.

Reply:
@Charles-Gagnon created https://github.com/microsoft/azuredatastudio/issues/6911 to address this,